functions/functions5.rs

```diff
@@ -40,5 +40,8 @@ fn square(num: i32) -> i32 {
 
 // This is a really common error that can be fixed by removing one character.
 // It happens because Rust distinguishes between expressions and statements: expressions return
-// a value and statements don't. We want to return a value from the `square` function, but it
-// isn't returning one right now...
+// a value based on its operand and statements simply retun a `()` type which just like `void` in C/C++ language. 
```

Comment:
i'd say `[...], and statements simply return a () type which behaves just like [...]` here